spec/lib/farscape/plugins_spec.rb

```diff
@@ -128,10 +136,21 @@ def call(env)
           saboteur_plugin = {name: :saboteur, type: :scarran, middleware: [saboteur_middleware]}
           detector_plugin = {name: :detector, type: :sebacean, middleware: [TestMiddleware::SabotageDetector]}
           [saboteur_plugin, detector_plugin].shuffle.each { |plugin| Farscape.register_plugin(plugin) }
+          
           expect(Farscape.middleware_stack.map{ |m| m[:class] }).to eq( [TestMiddleware::SabotageDetector, TestMiddleware::Saboteur] )
           expect{Farscape::Agent.new("http://localhost:#{RAILS_PORT}").enter}.not_to raise_error('Sabotage detected')
         end
       end
+      
+      it 'doesn\'t disable everything with one' do
+        detector_plugin = {name: :detector, type: :sebacean, middleware: [TestMiddleware::SabotageDetector], default_state: :disabled}
```

Comment:
Why you monkeywrenching!?

I was following the convention already established, but I can find/replace the names...